dts/bindings/pinctrl/nordic,nrf-pinctrl.yaml

```
@@ -46,6 +46,8 @@ description: |
     As shown, pin configurations are organized in groups within each child node.
     Each group can specify a list of pin function selections in the 'psels'
     property. The NRF_PSEL macro is used to specify a pin function selection.
+    If a pin needs to be explicitely disconnected, there is also the
```

```suggestion
    If a pin needs to be explicitly disconnected, there is also the
```
Also in the second modified description.